src/styles.css

```diff
@@ -0,0 +1,242 @@
+/* Global Styles */
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
+html {
+  box-sizing: border-box;
+  line-height: 1.15;
+  -webkit-text-size-adjust: 100%;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+
+  /* outline: 1px solid red !important; */
+}
+
+body {
+  --text: #414856;
+  --check: #4f29f0;
+  --disabled: #c3c8de;
+  --background: #fff;
+  --font: 'Montserrat', sans-serif;
+
+  background-color: #e8e8e8;
+  font-family: var(--font);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100vw;
+  height: 100vh;
+  padding: 3vh;
+}
+
+/* Text Styles */
+
+h1,
+p,
+button {
+  font-family: var(--font);
+}
+
+h1 {
+  font-size: 1.5rem;
+  font-weight: 100;
+  line-height: 1;
+  text-align: center;
+  letter-spacing: -8%;
+}
+
+p {
+  font-size: 1.15rem;
+}
+
+/* Checklist */
+
+#checklist {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: space-between;
+  background: var(--background);
+  border-radius: 0.5rem;
+  border: 1px solid rgba(0, 0, 0, 0.08);
+  box-shadow: 0 10px 30px rgba(65, 72, 86, 0.05);
+  max-width: 25rem;
+  width: 100%;
+  max-height: none;
+  transition: max-height 0s;
+}
+
+/* Header */
+
+#checklist .header {
+  width: 100%;
+  padding: 1.5rem 1rem 1rem 1rem;
+  align-self: flex-start;
+}
+
+/* Form */
+
+#add-form {
+  position: relative;
+  width: 100%;
+}
+
+/*
+#add-form label {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+*/
```

Comment:
Your code is very organized and clean I really liked how you used some effects to make it more visually engaging! I'd only say to kindly remember to delete commented code from your CSS file for better readability 👓

Reply:
I didn't notice this. Thank you very much Melanie =)